paver.sh

```diff
@@ -98,7 +98,13 @@ clean_build() {
     fi
     # In some case pip hangs with a build folder in temp and
     # will not continue until it is manually removed.
-    rm -rf /tmp/pip*
+    # On the OSX build server tmp is in $TMPDIR
+    if [ ! -z "${TMPDIR-}" ]; then
```

Comment:
Not sure what you mean… I tried the `bash` in the AIX 5.3 build slave, the oldest we've got:
```
-bash-3.2$ bash --version
GNU bash, version 3.2.16(1)-release (powerpc-ibm-aix5.2.0.0)
Copyright (C) 2005 Free Software Foundation, Inc.
-bash-3.2$ if [ -n "$NOSUCHDIR" ]; then echo 'we have $NOSUCHDIR!'; fi
-bash-3.2$ export NOSUCHDIR="/tmp"
-bash-3.2$ if [ -n "$NOSUCHDIR" ]; then echo 'we have $NOSUCHDIR!'; fi
we have $NOSUCHDIR!
```
I think using parameter expansion here is overkill and will confuse us later.